rudder-lang/src/opt.rs

```diff
@@ -57,4 +57,8 @@ pub struct Opt {
     /// Use json logs instead of human readable output
     #[structopt(long, short)]
     pub json_log: bool,
-}
\ No newline at end of file
+
+    /// Generate a backtrace in case an error occurs
```

Comment:
Why is it an option? Do you think people might prefer without backtrace?

Reply:
Backtrace is useful for rudder-lang contributors, but end users won't really make use of it